src/internal/context/cards-context.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
why is the new props passed through context? they're only passed one component level down, is there a reason why they aren't props?

Reply:
The initial idea was a bit more complicated, I'll clean up the context